hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1641,6 +1641,20 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.exception.stacktrace.propagate</name>
```

Comment:
Instead of making it an OM-specific setting, we could make it a global setting for all services.

Reply:
done the change in new PR
https://github.com/apache/hadoop-ozone/pull/12